src/containers/Ideas/Idea.js

```diff
@@ -366,80 +415,85 @@ export const Idea = (props) => {
             }}
             disabled={!isEditing}
           >
-            <View style={styles.titleContainer}>
-              <TextSourceSans
-                style={styles.title}
-              >
-                {ideaState.name}
-              </TextSourceSans>
-            </View>
-            <View style={styles.descriptionContainer}>
-              {ideaState.image && (
-                <>
-                  <Image
-                    accessibilityIgnoresInvertColors={true}
-                    source={{ uri: ideaState.image }}
-                    style={styles.ideaImage}
-                  />
-                </>
-              )}
-              <TextSourceSans
-                style={styles.text}
-              >
-                {ideaState.description}
-              </TextSourceSans>
-            </View>
-            {getLabels().length > 0 && (
-              <View style={styles.labelsContainer}>
-                {getLabels().map((label, idx) => (
-                  <Label key={idx + label} title={label} />
-                ))}
+            <View
+              ref={ideaRef}
+              collapsable={false}
```

Comment:
this "workaround"/"hack" is to prevent an issue with Android.

Described here:
https://github.com/facebook/react-native/issues/29712